build.xml

```diff
@@ -1090,6 +1111,35 @@ cd ..
     <copy todir="${lib.native}">
       <fileset dir="${build}" includes="${native.jar}"/>
     </copy>
+    <condition property="is_darwin">
+      <equals arg1="${os.prefix}" arg2="darwin"/>
+    </condition>
+    <property name="build.native.universal" value="${build}/native-darwin-universal"/>
+    <mkdir if:set="is_darwin" dir="${build.native.universal}/intel"/>
+    <mkdir if:set="is_darwin" dir="${build.native.universal}/aarch64"/>
+    <copy if:set="is_darwin" todir="${build.native.universal}/intel">
+      <zipfileset src="${lib.native}/darwin-intel.jar" includes="*jnidispatch*"/>
+    </copy>
+    <copy if:set="is_darwin" todir="${build.native.universal}/aarch64">
+      <zipfileset src="${lib.native}/darwin-aarch64.jar" includes="*jnidispatch*"/>
+    </copy>
+    <exec if:set="is_darwin" executable="lipo" dir="${build.native.universal}" failonerror="true">
+      <arg value="-create"/>
+      <arg value="-output"/>
+      <arg value="libjnidispatch.jnilib"/>
+      <arg value="intel/libjnidispatch.jnilib"/>
+      <arg value="aarch64/libjnidispatch.jnilib"/>
+    </exec>
+    <jar if:set="is_darwin" jarfile="${build}/${os.prefix}.jar" createUnicodeExtraFields="never" encoding="UTF-8">
+      <fileset dir="${build}/native-${os.prefix}-universal" includes="libjnidispatch.jnilib"/>
+      <manifest>
+        <attribute name="Implementation-Version" value="${jni.version} b${jni.build}"/>
+        <attribute name="Specification-Version" value="${jni.version}"/>
+      </manifest>
+    </jar>
```

Comment:
@matthiasblaesing This might be a bit crude. Please advise, if you'd prefer this e.g. as a `Makefile`, or if you have suggestions how to do this nicer in ant.